daml-script/runner/src/main/scala/com/daml/script/Runner.scala

```diff
@@ -311,8 +215,7 @@ class Runner(dar: Dar[(PackageId, Package)], applicationId: ApplicationId) exten
             }
             case SVariant(_, "AllocParty", v) => {
               v match {
-                case SRecord(_, _, vals) => {
-                  assert(vals.size == 2)
+                case SRecord(_, _, vals) if vals.size == 2 => {
```

Comment:
Should the exception we might throw below contain that we expected 2 fields in the record too? Same above.

Reply:
Good idea, fixed.